### PR TITLE
change contributing link

### PR DESCRIPTION
**Pull request by anikaluo**
I just changed the link to go to `CONTRIBUTING.md` in the repo. I think this is what BioJulia/Microbiome.jl#69 is asking for?

### CODE REVIEW

File: docs/src/contributing.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Contributing
 
 The BioJulia organisation has a set of contribution guidelines which apply to
-all BioJulia projects. These guidelines are available [here](http://biojulia.github.io/Contributing/latest) and it is
+all BioJulia projects. These guidelines are available [here](/CONTRIBUTING.md) and it is
```
Comment (kescobo) [inline docs/src/contributing.md:4]: I think the link in this file is just linking to itself, no? I think we want to link to https://github.com/BioJulia/BiobakeryUtils.jl/blob/main/CONTRIBUTING.md

Comment (kescobo) [reply docs/src/contributing.md:4]: Actually, perhaps this is a better source of truth: https://github.com/BioJulia/Contributing/blob/master/CONTRIBUTING.md

**Review by kescobo — COMMENTED** [review]
While we're at it, can you change line 11 to PR against the `main` branch instead of `master`, and   changing the line about gitter to 

> start a discussion in the `#biology` channel on [slack or zulip](https://julialang.org/community/), or using the [`biology` domain on discourse](https://discourse.julialang.org/c/domain/bio/).

File: docs/src/contributing.md (hunk 1 above)
Comment (anikaluo) [reply docs/src/contributing.md:4]: Okay got it. I linked it to the file in the main repo, which I think is pretty similar? But I can change the link again.

**Comment by anikaluo** [comment]
I can also do the same for Microbiome.jl. Can I get edit access to that repo please?

**Comment by kescobo** [comment]
Yes, though you shouldn't need it. The typical process is to fork, make a branch on your fork, then push. You can make a PR from there.

**Pull request merged**